From fd6575d663bf07d06143d0051ad7d8696f6c3d93 Mon Sep 17 00:00:00 2001
From: Alex Stewart <alex.stewart@ni.com>
Date: Tue, 23 Apr 2019 09:27:02 -0500
Subject: [PATCH] configure.ac: demote gacdir error to warn

The test for a valid gacdir may fail in cases where autoconfigure is run
outside of the build environment, as in openembedded builds. Demote the
error to a warning, because it is not always fatal.

Upstream-Status: Not Appropriate

Signed-off-by: Alex Stewart <alex.stewart@ni.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 72c29c196..d347ebdd2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -64,7 +64,7 @@ MONOGACDIR40="$MONOGACDIR"/4.0
 MONOGACDIR45="$MONOGACDIR"/4.5
 
 if ! test -e $MONOGACDIR20/mscorlib.dll -o -e $MONOGACDIR45/mscorlib.dll; then
-	AC_ERROR(Couldn't find the mono gac directory or mscorlib.dll in the usual places. Set --with-gacdir=<path>)
+	AC_MSG_WARN(Couldn't find the mono gac directory or mscorlib.dll in the usual places. Set --with-gacdir=<path>)
 fi
 
 if test -e $MONOLIBDIR/mono/xbuild-frameworks/.NETPortable/v4.0/Profile/Profile47/mscorlib.dll; then
-- 
2.11.0

